submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,272 @@
+# Burning Questions
+
+*by Sir Fendi*
+
+*version 1.1*
+
+(Calculations will be double-checked)
+
+Media:
+
+* Diagrams are included in a separate PowerPoint file. 
+
+* There is a preview image of the table as a PNG.
+
+All feedback is welcome! I am wondering:
+
+- Maybe more images can be added?
+
+- Do you think think the responses cover the questions sufficiently or maybe I have missed some factors to consider?
+
+- Are there any more questions to add?
+
+- Are there any other posts/explorations you have seen in the past that would be cool to mention here?
+
+. . . . . . . .
+
+Factorio has a variety of fuels that can go into your burner devices, ranging from classic and modest wood, up to sci-fi level nuclear fuel. The most popular burner fuel is coal, which is the backbone of early game automation and it runs all the boilers, furnaces, vehicles, and other machines reliably until it inevitably runs low. 
```

Comment:
Nuclear fuel doesn't make me think of sci-fi, although if you asked someone 50/100 years ago they'd definitely agree. The portable fusion reactor the player can put in their armor is definitely sci-fi, though. Or maybe you're calling it sci-fi because the player can have hundreds of them in their inventory or some other aspect of it that isn't realistic?

Reply:
Nuclear rocket fuel is mostly experimental IRL but yeah it is closer to reality than sci-fi, good point.

Reply:
> up to sci-fi level nuclear fuel

Are you not referring to the nuclear fuel that goes into nuclear reactors here? If not, I'd make that "nuclear rocket fuel".